draft-thomson-quic-tls.md

```diff
@@ -565,12 +593,8 @@ the maximum size of the TLS handshake.  This is unlikely to cause issues unless
 a server or client provides an abnormally large certificate chain.
 
 Stream 1 is exempt from the connection-level flow control window.
-
-
-### FEC Packets
-
-FEC packets MUST NOT be sent prior to completing the TLS handshake.  Endpoints
-MUST treat receipt of an unprotected FEC packet as a fatal error.
+TODO: Do we need this to be specified in this doc since it's already part
+of the main QUIC doc? No objection to keeping it here.
```

Comment:
It doesn't help to point out that the connection-level flow control won't be a problem in one way or other.  Note that stream 1 could be flow controlled, assuming that the connection-level initial value is the same or greater than the single stream value.